### PR TITLE
Updated endpoint/key formatting

### PR DESCRIPTION
**Pull request by wiazur**
Endpoints in Azure no longer rely on region alone, the new Azure resource accounts will use custom domains. Although region-based ones will still work. Also the formatting has been changed to always extract from environment variables.

### CODE REVIEW

**Comment by cthrash** [comment]
Since notebooks are interactive by nature, requiring users to set an environment variable seems like a cumbersome way to affect this change.  I think the wording could indeed change (that is, not use the term region) but I prefer the overall flow of how this used to be.  WDYT?

**Comment by wiazur** [comment]
Hi @cthrash , sure I understand the cumbersomeness (so to speak) of setting an env variable. After a lot of mistakenly published passwords and the like, @erhopf is leading the effort to change all samples/quickstarts to adhere to a standard. Erik could probably explain better about the new changes.

**Comment by erhopf** [comment]
@wiazur - for the notebook, I think we can leave this file as is. However, for normal .py files, I would like to make the change to env variables.

**Comment by wiazur** [comment]
Thanks for chiming in @erhopf . So I removed the environment variables and kept the endpoint generic, so users with new accounts can use their custom domain endpoints, as well as user with older region-based endpoints. Sound good, @cthrash ?

File: Jupyter Notebook/Computer Vision API Example.ipynb [hunk 1]
```diff
@@ -50,10 +50,9 @@
    },
    "outputs": [],
    "source": [
-    "# Variables\n",
-    "_region = 'westcentralus' #Here you enter the region of your subscription\n",
-    "_url = 'https://{}.api.cognitive.microsoft.com/vision/v2.0/analyze'.format(_region)\n",
-    "_key = None #Here you have to paste your primary key\n",
+    "# Variables\n"
+    "_url = <YOUR_ENDPOINT> # Here you have to paste your endpoint\n,
```
Comment (cthrash) [inline Jupyter Notebook/Computer Vision API Example.ipynb]: I don't like the fact that most of the URL is lost with this change.  I'd prefer that it looked something like:

```
_endpoint = 'YOUR_ENDPOINT' # Copy this from the Azure Portal; e.g. https://yourname.cognitiveservices.azure.com/
_url = '{}vision/v2.0/analyze'.format(_endpoint)\n",
```

Comment (wiazur) [reply Jupyter Notebook/Computer Vision API Example.ipynb]: Sure, that works. Yeah, I wasn't sure if you needed to append the service/version etc. Good call. That change works. Thanks. Also, I added edits to the OCR file, I left in the appendage, but change it more like this one if you want.

**Comment by erhopf** [comment]
@cthrash - Unfortunately, because of the way this feature was released, there is more than one endpoint that can be used with the service. The dev can either use the custom endpoint served up in the portal, or the existing regional endpoints for where their resource was created. In the long run, it's more manageable if we tell the user to grab the complete endpoint from the Azure portal rather than replace one piece of it. Then if for any reason, the platform team changes the url pattern, it's not hardcoded into the docs.

**Comment by wiazur** [comment]
@erhopf thanks Erik, good to know. I wasn't sure if the appendage added in this code was also the entire thing mirrored in the Azure portal. If that sounds good to Chris, I'll go ahead and make those changes, in a few minutes. Let me know if anything else looks amiss.

File: Jupyter Notebook/Computer Vision API Example.ipynb (hunk 1 above)
Comment (cthrash) [reply Jupyter Notebook/Computer Vision API Example.ipynb]: I'm a bit confused.  The portal only contains the base URL, so the old OCR notebook had it correctly.  I'd like to see the Compter Vision notebook to look like how OCR used to look like.
Screen Shot 2019-09-26 at 4 04 31 PM

Comment (wiazur) [reply Jupyter Notebook/Computer Vision API Example.ipynb]: True, I checked my Computer Vision resource and it's just the base. (Not all services provide base, but I guess some do). @erhopf Were you aware it's just the base URL for CV in Azure?

**Comment by erhopf** [comment]
I know that Chris H. was looking to standardize this across all services. With that said, there isn’t consistency across services with regard to what endpoint is provided.

**Comment by wiazur** [comment]
@erhopf Yeah, it is still inconsistent across services. But I thought the Azure endpoints were all moving towards the base URL? If so, it would be good to just leave in the appendages in this sample like we had before, no? That way the endpoint goal at least will be all consisent, and eventually in Azure they won't have the appendage anymore, it'll be handled in code or the SDK.
```
_endpoint = 'YOUR_ENDPOINT' # Copy this from the Azure Portal; e.g. https://yourname.cognitiveservices.azure.com/
_url = '{}vision/v2.0/analyze'.format(_endpoint)\n",
```

**Comment by erhopf** [comment]
I agree with your proposal. We should double check the portal before pushing the changes to reduce confusion for our devs.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by erhopf** [comment]
#sign-off

**Pull request merged**